lib/apps.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const ApiGroup = require('./api-group');
+
+class Apps extends ApiGroup {
+  constructor(options) {
+    const genericTypes = [
+      // Deprecated name of statefulsets in kubernetes 1.4
+      'petsets',
```

Comment:
Good point, my suggestion is premature. I've created #79 so that we don't lose it.